addons/im_livechat/data/im_livechat_chatbot_data.xml

```diff
@@ -6,10 +6,15 @@
         It's placed into 'data' to give them a starting point.
         From that record, they can duplicate / adapt / delete / ...
     -->
-
+    <record id="welcome_bot_operator_partner" model="res.partner">
+        <field name="name">Welcome Bot</field>
+        <field name="image_1920" type="base64" file="mail/static/src/img/odoobot.png"/>
+        <field name="active">False</field>
+    </record>
     <record id="chatbot_script_welcome_bot" model="chatbot.script">
         <field name="title">Welcome Bot</field>
         <field name="image_1920" type="base64" file="mail/static/src/img/odoobot.png"/>
+        <field name="operator_partner_id" ref="welcome_bot_operator_partner"/>
```

Comment:
To be checked with AJU but it feels like this whole file should be in demo and not in data?

Note: it's in noupdate so do we want an upgrade script to add it or not? Probably not, but it means old DB won't have the new bot here.

Shouldn't welcome_bot_operator_partner be created in demo and no need to link it to this channel?

If not moved, I would also avoid using this chatbot in demo and create another one instead.